constraint_utils/generic.py

```diff
@@ -34,6 +34,14 @@ def flip_base(base: hl.expr.StringExpression) -> hl.expr.StringExpression:
 
 
 def collapse_strand(ht: Union[hl.Table, hl.MatrixTable]) -> Union[hl.Table, hl.MatrixTable]:
+    """
+    Flips the base in 'ref', 'alt', 'context' columns if either 'G' or 'T' exists in 'ref' column.
+    
+    The reverse_complement_bases function has been made obsolete and will be replaced by reverse_complement.
+
+    param: ht: Input table
```

Comment:
```suggestion
    param: ht: Input Table.
```
(adding a period for consistency)